src/updater/installer.py

```
@@ -408,6 +408,6 @@ def _download_text(url: str) -> str:
         try:
             resp = httpx.get(url, follow_redirects=True, timeout=15.0)
             resp.raise_for_status()
-            return resp.text
+            return resp.text  # type: ignore[no-any-return]
```

Ruff (0.15.11)

[warning] Consider moving this statement to an `else` block

(TRY300)